### PR TITLE
add exclude flag isnt supported in ide

### PR DESCRIPTION
**Pull request by mirnawong1**
adding info the `exclude` flag isnt supported

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md [hunk 1]
```diff
@@ -98,9 +98,10 @@ The console section, located below the File editor, includes various console tab
 6. **Compiled Code tab &mdash;** The Compile button triggers a compile invocation that generates compiled code, which is displayed in the Compiled Code tab.
 <Lightbox src="/img/docs/dbt-cloud/cloud-ide/compiled-code-console-tab.jpg" width="100%" title="Compile results show up in the Compiled Code tab"/>
 
-7. **Lineage tab &mdash;** The Lineage tab in the File Editor displays the active model's lineage or  <Term id="dag" />. By default, it shows two degrees of lineage in both directions (`2+model_name+2`), however you can change it to +model+ (full DAG).
+7. **Lineage tab &mdash;** The Lineage tab in the File Editor displays the active model's lineage or  <Term id="dag" />. By default, it shows two degrees of lineage in both directions (`2+model_name+2`), however, you can change it to +model+ (full DAG).
     - Double-click a node in the DAG to open that file in a new tab
     - Expand the DAG and use node selection syntax (select or exclude) to view a subset of your DAG
```
Comment (patkearns10) [inline website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md]: Line 103 should probably change to exclude the exclude prompt!
from
```
- Expand the DAG and use node selection syntax (select or exclude) to view a subset of your DAG
```
to
```
Expand or shrink the DAG using node selection syntax
```

**Review by patkearns10 — CHANGES_REQUESTED** [review]
Otherwise, looks good!

**Comment by mirnawong1** [comment]
Nice catch @patkearns10 ! Have made that change, can you give it another look?

**Review by patkearns10 — APPROVED** [review]

**Comment by patkearns10** [comment]
🌵 LGTM!

**Pull request merged**